src/Components/Product.js

```diff
@@ -0,0 +1,22 @@
+import React
+    , {
+    useState
+} from "react";
+
+function Product({produce, emoticon, initialCount}) {
+    const [count, setCount] = useState(initialCount);
```

Comment:
Probleem is dat wanneer de initialCount veranderd (vanuit parent component) dit geen invloed heeft op de useState wanneer deze al is aangeroepen.

Om ervoor te zorgen dat je dan alsnog de waarde update moet je werken met een useEffect
```suggestion
    const [count, setCount] = useState(initialCount);
    
    useEffect(() => {
        setCount(initialCount);
    }, [initialCount]);
```